internal/provider/waypoint/resource_waypoint_template_test.go

```diff
@@ -175,6 +202,45 @@ resource "hcp_waypoint_template" "test" {
 }`, name)
 }
 
+func testTemplateWithActionsConfig(
+	templateName string,
+) string {
+	return fmt.Sprintf(`
+resource "hcp_waypoint_action" "test_create_action" {
+	name = "test_create_action"
+	description = "Test action"
+	request = {
+	    custom = {
+			method = "GET"
+			url = "https://example.com"
+			headers = {
+				Test-Header = "test"
+			}
+			body = "test"
+		}
+	}
+}
+
+resource "hcp_waypoint_template" "actions_template_test" {
+  name                     = "%s"
+  summary                  = "some summary for fun"
+  readme_markdown_template = base64encode("# Some Readme")
+  terraform_no_code_module_source = "private/waypoint-tfc-testing/waypoint-template-starter/null"
+  terraform_no_code_module_id = "nocode-7ZQjQoaPXvzs6Hvp"
+  terraform_project_id = "prj-gfVyPJ2q2Aurn25o"
+  terraform_cloud_workspace_details = {
+    name                 = "Default Project"
+    terraform_project_id = "prj-gfVyPJ2q2Aurn25o"
+  }
+  labels = ["one", "two"]
+  terraform_execution_mode = "remote"
+
+	actions = ["test_create_action"]
```

Comment:
nit, non-blocking: I'd format the indentation here. Sometimes I put this HCL in a .tf file and run `terraform fmt`, then copy the contents back here.

Reply:
Good catch! Will update now